### ISSUE TITLE
The Configuration Wizard creates a database for all profiles in the config file instead of the selected one

### ISSUE THREAD

**Issue by wondie** [gltn/stdm#92]
The Configuration Wizard creates a database for all profiles in the config file instead of the selected one. This is not desirable if the user is just working on a draft profile.

**Comment by gkahiu** [comment]
But you can always go back to the wizard and edit/extend the 'draft'
profiles and if one is not happy then they can always delete it coz at the
end of the day it is a 'draft'?

I do not see any implications of creating the database objects unless you
have specific reasons as to why it should not.

**Comment by wondie** [comment]
I get your point. However, my concern is, once the database tables are created, they can no longer edit entity names. This means, they have to go to the database and delete each entity table they want to edit. However, if we give the option of creating database table for the selected table only, there will be no such issue.

**Comment by gkahiu** [comment]
It will be contradictory because we are already advertising a list of other
profiles that the user can select and use right away so in this regard,
unless you are saying we now start having a criteria for choosing those
profiles which are 'draft' and those that are 'complete'. If it is draft,
will it be usable? No, because the database objects do not exist. If not,
should it be advertised in the list of available profiles? No, because it
is incomplete and the user will just get warnings of missing tables and
columns. So it is because is it like a work in progress that will be
completed later? If yes, then the user should create a skeleton framework
with the minimum STR defined and flesh it out with more details later.

As discussed, if its because of entity names then the only option is
removing and recreating. If we were using SQL Server then it allows for
editing table names but for PostgreSQL it is not an option.

**Comment by wondie** [comment]
That is true. So it is good to identify which profile is draft/incomplete with no database and which one is complete. And if it is incomplete with no database, not to add it in options profile list. 

I actually made a test on the impact of an draft/incomplete profile when running a complete profile. 
What I did was started a new profile. Then switched to a complete profile and run the wizard. The outcome was the wizard hangs.

**Comment by gkahiu** [comment]
I have not experienced a hanging wizard (probably more checks in the wizard
are needed if indeed the issue can be replicated) but at the minimum there
should be an STR defined as a 'draft' then changes can be made later.

My general opinion about this.

**Comment by wondie** [comment]
When I defined str, running the wizard was not a problem. It was only a problem when I created an empty profile. Are changes made in every page saved to the configuration.stc?
